index.js

```diff
@@ -73,7 +71,11 @@ const _splitToStream = (outStreamCreate, fileStream, partitionStreamSize, callba
 	});
 };
 
-const split = (fileStream, maxFileSize, rootFilePath, callback) => {
+const split = (fileStream, maxFileSize, rootFilePath, callback) => _split(fileStream, maxFileSize, rootFilePath, (r, n) => `${r}.split-${n}`, callback);
+
+const getSplitWithGenFilePath = (defaultGenerateFilePath) => ( (f, m, r, callback) => _split(f, m, r, defaultGenerateFilePath, callback) )
```

Comment:
Ahh, I just realized there's no point of giving the argument `r` to the generateFilePath here. Similarly, there would not be much value in supplying r to `_split` in this case if `generateFilePath` doesn't use it.

Reply:
You don't need to change this for the PR. I'll make the change in the future.